packages/hop-node/src/cli/swap.ts

```diff
@@ -121,19 +149,40 @@ async function main (source: any) {
 
     logger.debug(`attempting to swap ${l2Bridge.formatUnits(amountIn)} ${fromToken} for at least ${l2Bridge.formatUnits(minAmountOut)} ${toToken}`)
     tx = await amm.swap(fromTokenIndex, toTokenIndex, amountIn, minAmountOut, deadlineBn)
+
+    logger.info(`swap tx: ${tx.hash}`)
+
+    if (toNative) {
+      const receipt = await tx.wait()
+      for (const event of receipt.events) {
+        if (event.event === 'TokenSwap') {
+          logger.debug(`unwrapping ${toToken}`)
+          const parsedAmount = event.args.tokensBought
+          const unwrapTx = await unwrapToken(chain, parsedAmount)
+          logger.debug('waiting for unwrap tx confirmation')
+          await unwrapTx.wait()
+          tx = unwrapTx
+          break
+        }
+      }
+    }
   } else {
     logger.debug('uniswap swap')
```

Comment:
`uniswap` in the log can now be updated

Reply:
updated in https://github.com/hop-protocol/hop/commit/a38b614cc480ac4cfdd492433b6bb30041374d73